src/index.html

```diff
@@ -1,15 +1,175 @@
+          >
+        </div>
+        <div class="FieldBlock">
+          <label for="birthday">Full date of birth:</label>
+          <input
+          type="date"
+          id="birthday"
+          name="birthday"
+          >
+        </div>
+        <div class="FieldBlock">
+          <label for="agreement">I accept the term of the agreement</label>
+          <input
+          type="checkbox"
+          id="agreement"
+          name="agreement"
+          required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="FieldsBlock">
+        <legend>Registration</legend>
+        <div class="FieldBlock">
+          <label for="email">E-mail:</label>
+          <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </div>
+        <div class="FieldBlock">
+          <label for="password">Password:</label>
+          <input
+          type="password"
+          id="password"
+          name="password"
+          minlength="4"
+          maxlength="20"
+          required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="FieldsBlock">
+        <legend>An interesting fact about you!</legend>
+        <div class="FieldBlock">
+          <span>Do you love cats?</span>
+          <input
+          type="radio"
+          name="cat-lover"
+          id="yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+          type="radio"
+          name="cat-lover"
+          id="no"
+          >
+          <label for="yes">No</label>
+        </div>
+        <div class="FieldBlock">
+          <label for="color">What is your favorite color?</label>
+          <input
+          type="color"
+          name="favorite-color"
+          id="color"
+          >
+        </div>
+        <div class="FieldBlock">
+          <label for="sleeping">What time do you go to bed?</label>
+          <input
+          type="time"
+          name="sleeping"
+          id="sleeping"
+          >
+        </div>
+        <div class="FieldBlock">
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select
+          name="cars"
+          id="cars"
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+        <div class="FieldBlock">
+          <label for="rate">How do you rate our work</label>
+          <input
+          type="range"
+          name="rate"
+          id="rate"
+          min="1"
+          max="10"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="FieldsBlock">
+        <legend>Additional info:</legend>
+        <div class="FieldBlock">
+          <label for="comments">Comments:</label>
+          <textarea
+          name="comments"
+          id="comments"
+          >
```

Comment:
```suggestion
          <textarea
            name="comments"
            id="comments"
          >
```
все отлично только отредактируй все атрибуты подобным образом